WooCommerce/Classes/ViewRelated/Products/UpdateInventory/UpdateProductInventoryView.swift

```diff
@@ -0,0 +1,119 @@
+import SwiftUI
+import Combine
+import WooFoundation
+
+struct UpdateProductInventoryView: View {
+    @ObservedObject private(set) var viewModel: UpdateProductInventoryViewModel
+
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    @Environment(\.presentationMode) var presentationMode
```

Comment:
```suggestion
    @Environment(\.dismiss) private var dismiss
```

Then you can use `dismiss()` later on in the button. `presentationMode` was needed before iOS 15, but now we can use the clearer version, if it's just for dismissal.

Reply:
Yeah, done in 019fe664fe67366c8596d9e8dde08ed57837989c